api/src/main/java/io/druid/data/input/impl/CSVParseSpec.java

```diff
@@ -26,12 +26,14 @@
 import io.druid.java.util.common.parsers.Parser;
 
 import java.util.List;
+import java.util.Map;
 
 /**
  */
 public class CSVParseSpec extends ParseSpec
 {
   private final String listDelimiter;
+  private final Map<String, String> multiValueDelimiter;
```

Comment:
Probably should be `@Nullable`. Also applies to other fields and constructor parameters in this PR

Reply:
Yes, both listDelimiter and multiValueDelimiter should be @Nullable in constructor parameters

Reply:
Please also annotate fields, and parameters in other constructors (there is another constructor in this class), and also fields and all constructor parameters in other classes in this PR: `DelimitedParseSpec`, etc., where applies